### ISSUE TITLE
Allow from multiple networks generates invalid configuration file

### ISSUE THREAD

**Issue by mtb-xt** [PowerDNS/pdns_recursor-ansible#1]
Setting the allow_from as per the example (and the variable should be 'pdns_rec_config', not 'pdns_config' as in the example)
```
  vars:
    pdns_config:
      'allow-from':
        - '10.0.0.0/8'
        - '192.168.20.0/24'
```
generates the following config file -
```
allow-from=10.0.0.0/8
allow-from+=192.168.20.0/24
```
which is not accepted by the recursor:
```
NOTIFIED: [pdns_recursor-ansible | Restart PowerDNS Recursor] ***************** 
failed: [*.*.co.nz] => {"failed": true}
msg: Oct 12 00:50:47 Exception: Trying to set unexisting parameter 'allow-from+'
```

This line works - 'allow-from': '10.0.0.0/8, 192.168.20.0/24'

**Comment by pieterlexis** [comment]
Hi,

What version of the recursor are you running on what platform? This syntax is supported from 3.3.1 onward.

As for the example, will fix.

**Comment by mtb-xt** [comment]
Hi! 

I'm using it on Ubuntu 14.04.3 LTS, pdns-recursor 3.5.3-1.

**Comment by pieterlexis** [comment]
Ah, I misread, 3.3.1 is the auth, I'll see if I can make something nice to fix this on 14.04. In the meantime, just setting it as a string will work :).

**Comment by pieterlexis** [comment]
Hi,

Could you test the fixes in #4?

**Comment by mtb-xt** [comment]
It seems that #4 fixes the issue,
